frontend/src/App.vue

```diff
@@ -67,6 +67,18 @@ h1 {
   background-color: #f6f6f6;
 }
 
+// Apply styles only for monitor route
+.monitor {
```

Comment:
I think these would be better under the Monitor component — we can use global styles there.

Reply:
Resolved 🍻